spec/password.spec.js

```diff
@@ -0,0 +1,25 @@
+"use strict";
+
+describe("password", ()  => {
+    var password, cryptoFake;
+
+    beforeEach(() => {
+        cryptoFake = jasmine.createSpyObj("cryptoFake", [
+            "createHash",
+            "digest",
+            "update"
+        ]);
+        cryptoFake.createHash = jasmine.createSpy("crypto.createHash").andReturn(cryptoFake);
+        cryptoFake.digest = jasmine.createSpy("crypto.digest").andReturn("hashedcontent");
+        cryptoFake.update = jasmine.createSpy("crypto.update").andReturn(cryptoFake);
+        password = require("../lib/password")(cryptoFake);
+    });
+    it("hashes a passed in value", () => {
```

Comment:
Made to use Crypto instead of a mock of crypto and testing against hash passed in.